src/components/MultiHopTrade/components/TradeConfirm/ReceiveSummary.tsx

```diff
@@ -321,35 +305,7 @@ export const ReceiveSummary: FC<ReceiveSummaryProps> = memo(
             )}
           </Stack>
         </Collapse>
-        <Modal isOpen={showFeeModal} onClose={handleFeeModal} size='lg'>
-          <ModalOverlay />
-          <ModalContent>
-            <ModalCloseButton />
-            <Tabs variant='button'>
-              <TabList px={6} py={4} borderBottomWidth={1} borderColor='border.base'>
-                <Tab>{translate('foxDiscounts.feeSummary')}</Tab>
-                <Tab>{translate('foxDiscounts.simulateFee')}</Tab>
-              </TabList>
-              <TabPanels>
-                <TabPanel p={0}>
-                  <FeeBreakdown
-                    feeBps={shapeShiftFee?.affiliateBps ?? '0'}
-                    feeUserCurrency={shapeShiftFee?.amountAfterDiscountUserCurrency ?? '0'}
-                    foxDiscountPercent={shapeShiftFee?.foxDiscountPercent ?? '0'}
-                    feeBeforeDiscountUserCurrency={
-                      shapeShiftFee?.amountBeforeDiscountUserCurrency ?? '0'
-                    }
-                    feeDiscountUserCurrency={shapeShiftFee?.feeDiscountUserCurrency ?? '0'}
-                    feeBpsBeforeDiscount={shapeShiftFee?.potentialAffiliateBps ?? '0'}
-                  />
-                </TabPanel>
-                <TabPanel px={0} py={0}>
-                  <FeeExplainer borderRadius='none' bg='transparent' boxShadow='none' />
-                </TabPanel>
-              </TabPanels>
-            </Tabs>
-          </ModalContent>
-        </Modal>
+        <FeeModal isOpen={showFeeModal} onClose={handleFeeModal} shapeShiftFee={shapeShiftFee} />
```

Comment:
💜  lovely - was also going to extract it for the soon-to-be lending fees PR, so tyvm!